index.html

```diff
@@ -7,21 +7,72 @@
     <title>Awesome Books</title>
   </head>
   <body>
-    <h1>Awesome Books</h1>
-    <fieldset  id="list">
-      <legend>Books</legend>
-      <div class="black-border">
-        <ul id="booklist" class="flex-center">
+    <header class="header">
+      <nav class="nav">
+        <h2>Awesome Books</h2>
+        <ul class="menu">
+          <li> <a href="#">List</a> <hr></li>          
+          <li><a href="#add-new">Add New</a> <hr></li>
+          <li><a href="#contact">Contact</a></li>
         </ul>
-      </div>
-    </fieldset>
-    <hr class="underline"> 
-    <div id="add-new">
-        <h2>Add a new books</h2>
+      </nav>
+    </header>
+    <p class="time" id="date"></p>
+    <footer class="copy-right">
+      All Copyright reserved
+    </footer>
```

Comment:
- [ ] I think you need to place the footer element where it belongs, at the footer (down just before the </body> tag. Since you gave it a fixed position property the problem may not be noticed easily. Kindly make changes here. :pray:

Reply:
fixed